.github/workflows/focus-ios-ui-tests-previous-os.yml

```diff
@@ -165,12 +166,13 @@ jobs:
           sha: ${{ github.sha }}
       - name: Report to Slack (FullFunctionalTests)
         id: slack-fullfunctionaltests
-        uses: slackapi/slack-github-action@v1.26.0
+        uses: slackapi/slack-github-action@v2.0.0
         with:
           payload-file-path: ${{ env.browser }}/slack-fullfunctionaltests.json
+          payload-templated: true
+          webhook: ${{ secrets.WEBHOOK_SLACK_TOKEN }}
+          webhook-type: incoming-webhook
```

Comment:
Is this change due to the new version? I remember setting this options outside of env and did not work..

Reply:
Yes, `payload-templated`, `webhook` and `webhook-type` are necessary to have the Slack notifications to work the same as before. There are a number of breaking changes in v2 that we could not just bumped the version number.

Sample Slack notification using this version of `slack-github-action`: 
![Screenshot 2025-01-16 at 1 00 06 PM](https://github.com/user-attachments/assets/7a1ffd61-4edb-4767-afb4-e1222fd0978c)